runtime/test/particle-api-test.js

```diff
@@ -349,6 +349,225 @@ describe('particle-api', function() {
     await util.assertSingletonWillChangeTo(newView, Result, 'success');
   });
 
+  it('can load a recipe referencing a tagged handle in containing arc', async () => {
```

Comment:
These two test cases are further copy-paste-edit variants of the ones above in this file. As we discussed in https://github.com/PolymerLabs/arcs/pull/1204#discussion_r182291172 I think we can factor out some more generic helpers (less specific than I had done in that other PR) that just make it less verbose to start things up and then poke at the arc. Maybe we should add an explicit TODO and file an issue to do some of that soon and reference it here.